workspace/src/common/launch/art_control_launch/launch/art_control.launch.py

```diff
@@ -28,61 +28,54 @@
 # CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.#
+
+# ros imports
 from launch import LaunchDescription
-from launch.substitutions import LaunchConfiguration, TextSubstitution
-from launch.actions import DeclareLaunchArgument
 from launch_ros.actions import Node
 
+# internal imports
+from launch_utils import AddLaunchArgument, GetLaunchArgument
+
 
 def generate_launch_description():
-    launch_description = LaunchDescription()
+    ld = LaunchDescription()
 
     # ----------------
     # Launch Arguments
     # ----------------
 
-    def AddLaunchArgument(arg, default):
-        launch_description.add_action(
-            DeclareLaunchArgument(
-                arg,
-                default_value=TextSubstitution(text=default)
-            )
-        )
-
-    AddLaunchArgument("input/path", "/path_planning/path")
-    AddLaunchArgument("input/vehicle_state", "/vehicle/state")
-    AddLaunchArgument("output/vehicle_inputs", "/control/vehicle_inputs")
+    AddLaunchArgument(ld, "art_control/input/path", "/path_planning/path")
+    AddLaunchArgument(ld, "art_control/input/vehicle_state", "/vehicle/state")
+    AddLaunchArgument(ld, "art_control/output/vehicle_inputs", "/control/vehicle_inputs")
 
-    AddLaunchArgument("control_mode", "PID")
-    AddLaunchArgument("control_file", "data/smallest_radius_right.csv")
-    AddLaunchArgument("steering_gain", "1.6")
-    AddLaunchArgument("throttle_gain", "0.08")
-    AddLaunchArgument("use_sim_msg", "False")
-    AddLaunchArgument("use_sim_time", "False")
+    AddLaunchArgument(ld, "control_mode", "PID")
+    AddLaunchArgument(ld, "control_file", "data/smallest_radius_right.csv")
+    AddLaunchArgument(ld, "steering_gain", "1.6")
+    AddLaunchArgument(ld, "throttle_gain", "0.08")
+    AddLaunchArgument(ld, "use_sim_time", "False")
```

Comment:
use_sim_time is not specific to control... used in multiple launch files. Does this grab the correct variable?